### DISCUSSION TITLE
Proxmox GUI not working properly with application access

### DISCUSSION THREAD

**Discussion by tooboredtocode** [gravitational/teleport#6746] [Q&A] [answered]
I have recently started using Teleport to secure my cluster, which has been working nice so far.

Anyhow, some parts of the proxmox gui don't seem to be working anymore, and returning 501 errors. A proxmox employee told me to look at the full packets to see if Teleport modifies it somehow, leading to it not working.

I am now wondering, how i may do such thing and/or if anyone knows any solution.

(a link to the proxmox forum thread can be found [here](https://forum.proxmox.com/threads/proxmox-gui-not-working-properly.88608/))

**Comment by tooboredtocode** [comment, marked as answer]
So regarding this, adding an additional reverse proxy fixed it, which is really weird.

**Comment by webvictim** [comment]
This looks like an issue with cookies not being forwarded correctly or perhaps a cross-domain request issue. It's interesting that nginx fixes it, as it implies that Teleport's reverse proxying support is not fully-featured.

@r0mant For reference.

**Reply by tooboredtocode** [reply]
I don't know what is wrong, just that nginx somehow fixes it. Now I'm too afraid to touch it, fearing it'll break 😅
